samples/server/go-server/api/server/testing/testobjects.go

```diff
@@ -15,17 +15,20 @@
 package testutil
 
 import (
+	"fmt"
+	"log"
+
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/any"
 	pb "github.com/grafeas/grafeas/v1alpha1/proto"
 	opspb "google.golang.org/genproto/googleapis/longrunning"
-	"log"
 )
 
-func Occurrence(noteName string) *pb.Occurrence {
+func Occurrence(pID, noteName string) *pb.Occurrence {
+	name := fmt.Sprintf("projects/%s/occurrences/134", pID)
```

Comment:
Can you inline this below, since I think it is only used once?

Reply:
Fixed in 11c4c01